src/primordials.ts

```diff
@@ -0,0 +1,4 @@
+export const primordials: any = // TODO export as typeof import('node-primordials'), types are wrong
```

Comment:
this logic should probably be part of `node-primordials`. WDYT?

Reply:
Yeah, it'd be better to just update node-primordials so that it exports the desired object shape. Casting to `any` and hacking around it in this way defeats the benefits of TS. Happy to accept whatever changes to node-primordials will help facilitate that, it's still very malleable.